packages/msw-addon/src/mswDecorator.js

```diff
@@ -19,30 +21,33 @@ export function initialize(options) {
     server.listen(options)
     api = server
   }
+
   return api
 }
 
 export function getWorker() {
   if (api === undefined) {
-    throw new Error(`[MSW] Tried to get api but it was not defined yet. Did you forget to initialize it?`)
+    throw new Error(
+      `[MSW] Failed to retrieve the worker: no active worker found. Did you forget to call "initialize"?`
+    )
   }
 
-  return api;
+  return api
 }
 
 export const mswDecorator = (storyFn, { parameters: { msw = [] } }) => {
   if (api) {
-    api.resetHandlers();
+    api.resetHandlers()
```

Comment:
It looks like our code formatting setup is misconfigured, as we get different formatting output. Something to consider as a point of improvement.

Reply:
Definitely. We should add prettier and husky to make sure all files are formatted the same way. I could do that in a separate PR!